tests/tape_tests/test_ensemble_frame.py

```diff
@@ -0,0 +1,89 @@
+""" Test EnsembleFrame (inherited from Dask.DataFrame) creation and manipulations. """
+import pandas as pd
+from tape import Ensemble, EnsembleFrame, TapeFrame
+
+import pytest
+
+# Create some fake lightcurve data with two IDs (8001, 8002), two bands ["g", "b"]
+# and a few time steps.
+SAMPLE_LC_DATA = {
+        "id": [8001, 8001, 8001, 8001, 8002, 8002, 8002, 8002, 8002],
+        "time": [10.1, 10.2, 10.2, 11.1, 11.2, 11.3, 11.4, 15.0, 15.1],
+        "band": ["g", "g", "b", "g", "b", "g", "g", "g", "g"],
+        "err": [1.0, 2.0, 1.0, 3.0, 2.0, 3.0, 4.0, 5.0, 6.0],
+        "flux": [1.0, 2.0, 5.0, 3.0, 1.0, 2.0, 3.0, 4.0, 5.0],
+    }
+TEST_LABEL = "test_frame"
+TEST_ENSEMBLE = Ensemble()
```

Comment:
Similar recommendation to use a fixture, and additionally anytime we test an Ensemble we should use the client fixture established here: https://github.com/lincc-frameworks/tape/blob/main/tests/tape_tests/conftest.py#L10

Without it Dask will spin up multiple clusters which can cause a lot of needless warnings with unit testing.

Reply:
Done.